### PR TITLE
[exporter/tanzuobservability] Bump to wavefront-sdk-go@v0.10.1

### PR DESCRIPTION
**Pull request by keep94**
**Description:** <Describe what has changed.>
Depend on github.com/wavefronthq/wavefront-sdk-go@v0.10.1

**Testing:** 
Unit tests and end-to-end integration tests.

### CODE REVIEW

File: exporter/tanzuobservabilityexporter/metrics_exporter.go [hunk 1]
```diff
@@ -64,11 +61,10 @@ func newMetricsExporter(settings component.ExporterCreateSettings, c config.Expo
 	if !cfg.hasMetricsEndpoint() {
 		return nil, fmt.Errorf("metrics.endpoint required")
 	}
-	hostName, port, err := cfg.parseMetricsEndpoint()
-	if err != nil {
+	if _, _, err := cfg.parseMetricsEndpoint(); err != nil {
```
Comment (TylerHelmuth) [inline exporter/tanzuobservabilityexporter/metrics_exporter.go:64]: Since `cfg.Metrics.Endpoint` is being used now is this function still necessary?

**Review by TylerHelmuth — COMMENTED** [review]
does exporter_test.go needs any updates?

File: cmd/configschema/go.mod [hunk 2]
```diff
@@ -7,7 +7,7 @@ require (
 	github.com/google/uuid v1.3.0
 	github.com/open-telemetry/opentelemetry-collector-contrib v0.54.0
 	github.com/open-telemetry/opentelemetry-collector-contrib/pkg/resourcetotelemetry v0.54.0
-	github.com/stretchr/testify v1.7.4
+	github.com/stretchr/testify v1.7.5
```
[the remark this answers is not in the document]
Comment (keep94) [reply cmd/configschema/go.mod:10]: Done.

File: exporter/tanzuobservabilityexporter/exporter.go [hunk 3]
```diff
@@ -70,7 +70,7 @@ func newTracesExporter(settings component.ExporterCreateSettings, c config.Expor
 	if !cfg.hasTracesEndpoint() {
 		return nil, fmt.Errorf("traces.endpoint required")
 	}
-	tracingHost, tracingPort, err := cfg.parseTracesEndpoint()
+	_, _, err := cfg.parseTracesEndpoint()
```
[the remark this answers is not in the document]
Comment (keep94) [reply exporter/tanzuobservabilityexporter/exporter.go:73]: Good catch. This parseTracesEndpoint() function doesn't need to be called here.

File: exporter/tanzuobservabilityexporter/metrics_exporter.go (hunk 1 above)
Comment (keep94) [reply exporter/tanzuobservabilityexporter/metrics_exporter.go:64]: This call to parseMetricsEndpoint() is no longer needed.

Comment (keep94) [reply exporter/tanzuobservabilityexporter/metrics_exporter.go:64]: Actually when I took this code out, tests started failing. We expect a port number to be included with the URL, and parseMetricsEndpoint checks for that.

File: exporter/tanzuobservabilityexporter/exporter.go (hunk 3 above)
[the remark this answers is not in the document]
Comment (keep94) [reply exporter/tanzuobservabilityexporter/exporter.go:73]: When I took out this code, tests started failing, so I added it back.

**Comment by TylerHelmuth** [comment]
@keep94 do you want me to reopen this PR?

**Pull request closed without merging**